flow/Platform.actor.cpp

```diff
@@ -2780,6 +2780,14 @@ void deprioritizeThread() {
 #endif
 }
 
+void setThreadPriority(int pri) {
```

Comment:
This is almost the same as `deprioritizeThread`. can you remove `deprioritizeThread`?

Reply:
Change applied.